backend/infrastructure/llm/providers/anthropic.py

```
@@ -255,3 +237,242 @@ def _parse_response(self, raw: dict) -> ChatResponse:
+            messages: チャットメッセージ履歴
+            tools: 利用可能なツール
+            temperature: 生成の多様性
+            max_tokens: 最大トークン数
+
+        Yields:
+            StreamChunk: 各ストリーミングチャンク
+
+        Raises:
+            httpx.HTTPError: API呼び出しに失敗した場合
+        """
+        # systemメッセージを分離（複数ある場合は結合）
```

Ruff (0.14.13)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)